index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css">
     <link rel="stylesheet" href="calendar.css"/>
 </head>
-<body>
+<body onload = "displayCalendar()">
```

Comment:
remove this and use an immediately invoked function expression in your code.

Reply:
There are two separate functions that need to be invoked based on which URL is being loaded. That is the reason I added this function. Let me know if there is an alternative.

Reply:
there must be 1 html page per url. Please don't use 1 html file for everything.